pkg/registry/core/pod/strategy.go

```diff
@@ -145,6 +145,9 @@ func (podStrategy) CheckGracefulDelete(ctx context.Context, obj runtime.Object,
 		// use the default value if set, or deletes the pod immediately (0)
```

Comment:
should we fix this comment to explain that if value 0 it will use 1?

Reply:
Also update the comment for `pod.Spec.TerminationGracePeriodSeconds` in types.go